zwave_js_server/model/controller/__init__.py

```diff
@@ -261,6 +261,10 @@ def update(self, data: ControllerDataType) -> None:
                 self.data["rebuildRoutesProgress"]
             )
 
+    def update_inclusion_state(self, state: InclusionState) -> None:
```

Comment:
Update the event handler `handle_inclusion_state_changed` instead.

Each incoming event should have a corresponding event handler in the appropriate model. The event handler should update state as needed before emitting the event, calling event listeners of the event. This is how the library is designed.

Reply:
https://github.com/home-assistant-libs/zwave-js-server-python/pull/1173